tests/robustness/validate/validate.go

```diff
@@ -35,11 +35,12 @@ func ValidateAndReturnVisualize(t *testing.T, lg *zap.Logger, cfg Config, report
 	linearizableOperations := patchLinearizableOperations(reports, persistedRequests)
 	serializableOperations := filterSerializableOperations(reports)
 
-	linearizable, visualize := validateLinearizableOperationsAndVisualize(lg, linearizableOperations, timeout)
+	linearizable, results := validateLinearizableOperationsAndVisualize(lg, linearizableOperations, timeout)
 	if linearizable != porcupine.Ok {
 		t.Error("Failed linearization, skipping further validation")
-		return visualize
+		return results.Visualize
```

Comment:
Why not return results struct? It should allow caller to call results.Visualize and futher eliminate usage of arbitrary function interface.

Reply:
Yeah right. Will do it.